README.md

```diff
@@ -56,14 +56,25 @@ Each user is allowed to have different local specifications: We don't put any re
 
 ##### What do I put in `user-config.yaml`?
 
-There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Two things we do recommend keeping in `user-config.yaml` are the name of your Stata executable—which we'll prompt you to enter if necessary—and the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html)—if you have one. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directory named cache/template on Dropbox would be 
+There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Three things we do recommend keeping in `user-config.yaml` are the name of your Stata executable, the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html), and the location of a release directory. These fields don't have to be specified if you're not using them, and we'll prompt you for their values at runtime if you've forgotten to specify them and they're necessary. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directories named cache/template and release on Dropbox would be 
 
 ```YAML
-stata_executable: statamp
+stata_executable: StataMP
 cache_directory: /Users/Example_User/Dropbox/cache/template
 release_directory: /Users/Example_User/Dropbox/release/
 ```
 
+##### How do I handle data external to my repository?
+
+We are agnostic about how you incorporate external data into the template. There's no custom builder for these assets, by design. We expect data storage and transfer protocols to differ—with good reason—across projects and labs, and we try to keep the template flexible enough for any use. That said, we do have some suggestions:
+
+* When a large dataset is stored locally, `user-config.yaml` can include an entry specifying the user-specific path to that dataset. The key of the entry should be constant across users and documented in the top-level readme of the repository. Scripts can access the path to the dataset by parsing `user-config.yaml` and referencing the key. 
```

Comment:
I'd cut "Scripts can access the path to the dataset by parsing `user-config.yaml` and referencing the key.". That's restating the purpose of `user-config.yaml` and we already said that upstairs.